typings/solc.d.ts

```diff
@@ -1,7 +1,7 @@
 declare module 'solc' {
     interface CompilerInputSourceFile {
         keccak256?: string;
-        urls: string[];
+        content: string;
```

Comment:
It looks like I got the definition here wrong, _either_ `urls` or `content` is allowed so the definition should be:
```
interface CompilerInputSourceFile {
    keccak256?: string;
    urls: string[];
}
interface CompilerInputSourceCode {
    keccak256?: string;
    content: string;
}
interface CompilerInput {
    language: "Solidity" | "serpent" | "lll" | "assembly";
    sources: {
        [globalName: string]: CompilerInputSourceFile|CompilerInputSourceCode,
    }
}
```

The key to this working is the union type on the `sources` object values.  It is saying that for any key in `sources`, the value can be _either_ a `CompilerInputSourceFile` or a `CompilerInputSourceCode` object.

Reply:
Oops, yeah I knew it was the case that either urls or content is allowed, but I didn't catch this either.  Will update this.